UniversalDashboard.Forge.psm1

```diff
@@ -117,36 +117,44 @@ function New-UDDesktopApp {
         if ($PathInfo.PSIsContainer)
         {
             Write-Verbose "Copying contents of $Path to $src"
-            Copy-Item -Path "$($PathInfo.FullName)/*" -Destination $src -Container -Recurse
+            Copy-Item -Path "$($PathInfo.FullName)/*" -Destination $src -Exclude (Split-Path $OutputPath -Leaf) -Recurse
```

Comment:
I had a particular test case where if I used an output path that existed inside the folder where my `dashboard.ps1`  existed `Copy-Item` went a bit crazy. Since a non-existent directory is created on copy when the `-Recurse` switch is used `Copy-Item` would go down a rabbit hole and endlessly create a nested output path directory structure. I added the `-Exclude` to account for that.  I also removed the `-Container` because the help for `Copy-Item' says that parameter is true by default (which kind of defeats the purpose of a switch but yea...).